main/networks/list.cjs

```diff
@@ -150,7 +147,7 @@ module.exports =
         },
         "startBlockV1": 14820000,
         "logsQueryRange": 20000,
-        "explorer": "https://blockscout.com/xdai/mainnet",
+        "explorer": "https://gnosisscan.io/",
```

Comment:
we seem to not like trailing slashes, remove here too?

Reply:
sure